zebrad/src/components/mempool/downloads.rs

```diff
@@ -124,7 +124,7 @@ pub enum TransactionDownloadVerifyError {
     #[error("transaction download / verification was cancelled")]
     Cancelled,
 
-    #[error("transaction did not pass consensus validation: {0}")]
+    #[error("transaction did not pass consensus validation, error: {0:?}")]
```

Comment:
Optional: revert if you agree with the reasoning here https://github.com/ZcashFoundation/zebra/pull/9066#discussion_r1869582725

```suggestion
    #[error("transaction did not pass consensus validation: {0}")]
```